spec/views/layouts/_x_edit_buttons.html.haml_spec.rb

```diff
@@ -0,0 +1,14 @@
+describe 'layouts/_x_edit_buttons.html.haml' do
+  context 'policy simulation' do
+    let(:record) { FactoryBot.create(:vm_infra) }
+
+    before { assign(:record, record) }
+
+    it 'renders Back button with proper onclick' do
+      render :partial => 'layouts/x_edit_buttons',
+             :locals  => {:record_id => record.id,
+             	          :params    => {:action => 'policies'}}
+      expect(response).to include('title="Back" onclick="miqAjaxButton(&#39;/vm_infra/policy_sim?continue=true&#39;);">')
```

Comment:
Have you tried escaping the `"` with `\"` instead of `&#39;`?

Reply:
Yes, of course, but it did not work for some reason.I will try again to solve this.

Reply:
That's enough for me :+1: